flutter-idea/src/io/flutter/run/daemon/DevToolsService.java

```diff
@@ -123,7 +124,8 @@ private void startServer() {
                                       ImmutableList.of("--machine")));
         }
         else {
-          setUpWithDart(createCommand(DartSdk.getDartSdk(project).getHomePath(), "dart", ImmutableList.of("devtools", "--machine")));
+          setUpWithDart(createCommand(DartSdk.getDartSdk(project).getHomePath(), DartSdk.getDartSdk(project).getHomePath() + File.separatorChar + "bin" + File.separatorChar + "dart",
```

Comment:
This does work on Windows since the file has the same name, and I don't see a Dart plugin function. I'll merge this as is for now but maybe @jwren has a suggestion for improvement here?

Reply:
This does work on Windows since the file has the same name, and I don't see a Dart plugin function. I'll merge this as is for now but maybe @jwren has a suggestion for improvement here?